GurshanBrar.md

```diff
@@ -0,0 +1,39 @@
+---
+name: "@GurshanBrar"
+project: "3d Printed Prosthetic Hand"
+---
+
+# Project Name
+3D Printed Prosthetic Hand
+## Summary
+What are you going to build? What does it do? Why are you excited to build it?
+
+For our winter project, we are going to build a 3d printed prosthetic hand. The hand will include parts for the joints of the fingers, the palms, and the thumb. The hand will feature opening and closing, and will have opportunity for more features due to the construction. We want to build this hand because it is a very lightweight, affordable solution for people with disabilities. Because it is 3d printed, it takes very minimal resources to create and is very adaptable. This product could be sold at a very cheap price to low income individuals as well as regular people. 
+## Plan
+
+What steps are you going to take to build it?
+1. First we are going to collect the arduino, servos, battery pack, wires, and high strength wire. 
+2. Then we’ll print the hand segments and all the plastic parts
+3. We’ll put the hand together and put the servos, battery inside
+4. We’ll connect everything with the wire
+5. We’ll link up the impulse sensors
+6. We’ll use arduino to code the servo movements with the sensors
+
+What tools are you going to use? 
+
+To build the project, we’ll use a 3d printer, pliers, soldering iron, screwdriver, a needle, a scraper.
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Servos  | https://www.amazon.com/ZOSKAY-Coreless-Digital-Stainless-arduino/dp/B07S9XZYN2/ref=sr_1_5?crid=1NS096T66DA1V&keywords=servos&qid=1672550781&sprefix=servo%2Caps%2C178&sr=8-5 | Qt 2: $64.00  |
+| Li-Po Battery | https://www.amazon.com/Zeee-Batteries-Dean-Style-Connector-Vehicles/dp/B076Z778MJ/ref=sr_1_3?crid=1OZ7E6YQFJK7K&keywords=7+volt+li+po&qid=1672551064&sprefix=7+volt+li+po%2Caps%2C137&sr=8-3 | $31.19 |
```

Comment:
That's a good point, I think we overlooked that. We can use a separate battery for the arduino, this would be better for testing.

Reply:
@GurshanBrar ok cool, do you already have access to that battery?

Reply:
also, i'm wondering, are you following any guides of sort? also, where will you be placing the servos? i noticed you are only purchasing 2? could you please add a little more details on what your end product would look like and how it would work? thank you sm!

Reply:
My bad, I forgot to update the table. So we've decided that instead of getting another battery, we'll use the arduino uno and use the same battery for powering the arduino. This is a better option because the arduino uno has more processing power and that way we only need to charge one battery. The servos will be going inside the casing of the hand. 

The construction of the hand is like this: 
1) The hand consists of 1 palm piece and 1 piece for the top.
2) Each finger is consisting of 3 individual "joints". The thumb also consits of these joint pieces.
3) the joint pieces are connected by fishing line. When you pull the fishing line, the hand comes together.
4) One of the servos is used for the 4 fingers. We pull fishing line through each finger, and then tie that together into one node. The servo controls that node, pulling all 4 fingers at once.
5) The second servo controls the thumb, using fishing line.
6) The end product is a hand that attaches to your wrist. It has impulse sensors connecting to the forearm.

As for our guide, we'll be watching YouTube videos about 3d printed prosthetic hands to start off. Then we'll model and test out our own variation to make the best solution. For the arduino and wiring, we'll find arduino tutorials on how to control servos, and how to receive inputs through the impulse sensors.

Hope this clears things up , thanks!

Reply:
thank you so much! can you please confirm that this battery is compatible with your servos and arduino (ie. it'll actually be able to power both)?

and, this breakdown is great, could you please add it under plan? 

after which we should be good to move forward! thank you for your patience :)

Reply:
So the battery is 7.4v. The arduino has a voltage regulator that regulates 7-12v, and the servo's voltage range is 5v to 7.4v. So voltage is good, and the battery has 5200 MAH of current, which is more then enough to power the two servos and arduino.

I'll add it under my plan, thanks!